contract/src/swaparoo.contract.js

```diff
@@ -146,9 +155,11 @@ export const start = async (zcf, privateArgs, baggage) => {
         Object.keys(want).map(k => [k, M.any()]),
       );
 
-      return M.splitRecord({
-        give: M.splitRecord(givePattern),
-      });
+      return harden(
```

Comment:
doesn't `splitRecord` return a hardened object?

Reply:
You are right. It does. Will fix.